package.json

```diff
@@ -10,13 +10,16 @@
   "scripts": {
     "dev": "nuxt",
     "build": "nuxt build",
+    "build:local": "rimraf fsxa-pwa-*.tgz && npm version prerelease --no-git-tag-version --preid alpha && npm run build && npm pack",
     "generate": "nuxt generate",
     "start": "nuxt start",
     "lint": "eslint --ext .js,.vue,.ts,.tsx --ignore-path .gitignore .",
     "test": "jest",
     "commit": "git-cz",
     "commitmsg": "commitlint -E HUSKY_GIT_PARAMS",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "cy:open": "cypress run test",
```

Comment:
Under the script name `cy:open` I would assume that you would run `cypress open` and not `cypress run`. 
Maybe you could change this and add another script. So have would have `open` and `run` with two different scripts.